core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/dataformats/DataFormatModel.java

```diff
@@ -72,7 +72,11 @@ public void setSupportedDataFormats(ArrayList<DataFormat> supportedDataFormats)
 	 * @return
 	 */
 	public DataFormat getDataFormatByName(String dataformat) {
-		return supportedDataFormats.stream().filter(df -> df.getName().equals(dataformat)).findFirst().orElse(null);
+		DataFormat foundDF = supportedDataFormats.stream().filter(df -> df.getName().equals(dataformat)).findFirst().orElse(null);
+		if (foundDF == null) {
+			foundDF = supportedDataFormats.stream().filter(df -> df.getModelName().equals(dataformat)).findFirst().orElse(null);
```

Comment:
I think we're relying on the "name" or "model name" being unique vs. the GAV. But I'm just basing that assumption on the functionality that was there previously.